framework/console/controllers/BaseMigrateController.php

```diff
@@ -41,6 +41,18 @@
      * or a file path.
      */
     public $templateFile;
+    /**
+     * @var array the template file for generating migration code automatically.
```

Comment:
another question, drop_table should be generate primary key in up method  like create_table?